.github/workflows/github-actions.yml

```diff
@@ -73,8 +73,7 @@ jobs:
       - name: Build legateboost docs
         working-directory: docs
         run: |
-          # invoke sphinx build using legate
-          legate --run-mode=python --module sphinx.cmd.build source build -b html
+          make html
```

Comment:
I wasn't sure why `legate` was being used to run the Sphinx build. I'm hoping it's ok to use the Sphinx `Makefile` instead, so we can customize the build in the way this PR proposes.

Reply:
I remember not being able to get it to work the normal way, but if it works for you then that would be great :)

Reply:
Yep it's been working!

Wonder if maybe having legate packages (#113) + building in a conda environment prevents whatever issues you were seeing before.